core/src/i18n/en.yaml

```diff
@@ -166,15 +167,27 @@ certificates:
   secret-ref: Secret to use
   create:
     messages:
-      failure: Failed to create the certificate.
+      failure: Failed to create the Certificate.
       success: Certificate created.
+  messages:
+    decode-error: 'Failed to decode the CSR: {{message}}.'
   placeholders:
     common-name: Certificate CN (max 64 characters)
     csr: Plain text Certificate Signing Request
     dns-names: Additional domain names, one per line
     encoded-csr: Base64-encoded Certificate Signing Request
     name: Certificate name
     secret-name: Requested Secret name
+  tooltips:
+    common-name: Common Name is the primary domain name of the Certificate.
+    csr: Certificate Signing Request is an alternative way to provide Common Name, DNS Names, and other information.
+    with-csr: Enable to use a Certificate Signing Request instead of providing domain names manually.
+    dns-names: DNS Names are the optional additional domain names of the Certificate.
+    renew: Trigger a certificate renewal if enabled.
+    issuer: Certificate Issuer. If no issuer is given, default issuer will be used.
+    existing-secret: Use an existing Secret for storing the certificate if enabled. Otherwise a new Secret will be created.
```

Comment:
Yep, Secret is indeed a word that we [capitalize](https://kyma-project.io/community/guidelines/content-guidelines/08-style-and-terminology#capitalized-terms)! :)